SGMAcquaman_tests.pro

```diff
@@ -14,9 +14,9 @@ HEADERS += \
     source/tests/SGM/SGMEnergyPositionTest.h \
     source/tests/SGM/SGMEnergyControlTestView.h \
     source/tests/SGM/SGMGratingVelocityTests.h \
-    source/tests/SGM/SGMUndulatorVelocityTests.h \
-    source/tests/SGM/AMScalerAnalysisTest.h \
-    source/tests/SGM/AMScalerTestData.h
+    source/tests/SGM/SGMUndulatorVelocityTests.h #\
+    #source/tests/SGM/AMScalerAnalysisTest.h \
+    #source/tests/SGM/AMScalerTestData.h
 
```

Comment:
@iainworkman and @dretrex 
I removed these because I noticed that a lot of builds really struggled on these files. I think they're really big (and fully of text?) ... I saw at least one case where the OPI hung up trying to do this compile.